CONTRIBUTING.md

```diff
@@ -280,3 +286,17 @@ When a document is received and written by the connector, you'll see logging lik
 ```
 [2018-12-20 12:54:13,561] INFO flushing 1 queued docs (com.marklogic.client.datamovement.impl.WriteBatcherImpl:549)
 ```
+
+## Source Connector Testing
+To test out the source connector, you can insert documents into MarkLogic that contain data that matches TDE templates
+to generate Optic rows that match your query. For example, if your TDE template defines an "Authors" view in the
+"Medical" schema, the source configuration file might have the following properties.
+
+    ml.source.optic.dsl=op.fromView("Medical", "Authors")
+    ml.source.topic=Authors
+
+Then, after the connector is running, monitor the "Authors" topic. When a document is retrieved from MarkLogic and
```

Comment:
How does the user actually monitor the topic? Is there an equivalent process to bin/kafka-console-producer.sh that the user can use to see what's in a topic? I don't think the logging will suffice because the user primarily wants to see what the data looks like that ends up in a topic. A CP user can do that view the "Messages" tab for a topic. 

I just saw there's a kafka-console-consumer.sh tool - that sounds like it would do the job and is what we'd then want to mention here.